tests/vmi_multus_test.go

```diff
@@ -625,28 +632,29 @@ var _ = Describe("[Serial]SRIOV", func() {
 	})
 
 	Context("VirtualMachineInstance with sriov plugin interface", func() {
-		getSriovVmi := func(networks []string) *v1.VirtualMachineInstance {
-			// Pre-configured container-disk image for sriov-lane
-			vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskFedoraSRIOVLane))
 
-			// fedora requires some more memory to boot without kernel panics
-			vmi.Spec.Domain.Resources.Requests[k8sv1.ResourceName("memory")] = resource.MustParse("1024M")
-
-			// newer fedora kernels may require hardware RNG to boot
-			vmi.Spec.Domain.Devices.Rng = &v1.Rng{}
-
-			// pod network interface
-			vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{{Name: "default", Ports: []v1.Port{}, InterfaceBindingMethod: v1.InterfaceBindingMethod{Masquerade: &v1.InterfaceMasquerade{}}}}
-			vmi.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
+		getSriovVmi := func(networks []string, cloudInitNetworkData string) *v1.VirtualMachineInstance {
 
+			withSriovOptions := []libvmi.Option{
```

Comment:
No SRIOV options here yet, the name is confusing.
How about just `vmiOptions` ?

Reply:
done renamed to `withVmiOptions` so we have `NewSriovFedora(withVmiOptions)`